src/style.css

```
@@ -1 +1,34 @@
-/* write your styles here */
+body {
+  margin: 0;
+}
+
+div {
+  width: 300px;
+  height: 100px;
+  margin: auto;
```

Вертикальні відступи необов'язкові
```suggestion
  margin: 0 auto;
```